test/test_transformer.py

```diff
@@ -66,6 +68,49 @@ def get_layers(module):
     return layers
 
 
+def pipe_to_sd(pipe):
+    sd = {}
+    for stage_idx in range(pipe.num_stages):
+        stage_mod = pipe.get_stage_module(stage_idx)
+        sd[f"stage_{stage_idx}"] = stage_mod
+    return sd
+
+with tempfile.TemporaryDirectory() as tmpdir:
+    #Simulate saving the pipe
+    # Option 1:
```

Comment:
I think Option 1 would be more likely used than Option 2 in realistic setting. Could you please uncomment this block of code?